### DISCUSSION TITLE
Add shortcut in vs code

### DISCUSSION THREAD

**Discussion by MarkKarter** [community/community#69451] [Programming Help]
### Body

hi,
can you tell me how to add short word to generate react function code snippet?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by 0-mayurkaretha** [comment]
1. Go to File -> Preferences -> Configure User Snippets
2. Select "New Global Snippets file" and type <any-name> you want and hit enter
3. File should be created with the following name <any-name>.code-snippet
4. paste this code
  `{`
  `  "React Functional Component": {`
    `    "prefix": "rfce",`
    `    "body": [`
      `        "import React from \"react\"",`
     `         "",`
    `        "const ${1:name} = (props) => {",`
      `    "  return (",`
     `    "    <div>",`
      `    "      $2",`
     `     "    </div>",`
     `   "  )",`
    `    "};",`
     `     "",`
     `    "export default ${1:name};",`
    `     ""`
   `     ],`
   ` "description": "React Functional Component"`
 ` }`
`}`

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
